### DISCUSSION TITLE
The remote server desktop always displays the screen, which is not very secure

### DISCUSSION THREAD

**Discussion by heshen** [rustdesk/rustdesk#640] [General]
The remote server desktop always displays the screen, which is not very secure.
How to make the remote desktop not display the virtual desktop login like rdp, lock the server desktop and not display the content

远程的服务端桌面始终显示画面，不太安全。
如何让远程的桌面不显示类似rdp的虚拟桌面登录，锁定服务端桌面，不显示内容

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/65
